src/main/java/uk/gov/justice/digital/delius/transformers/AppointmentPatchRequestTransformer.java

```diff
@@ -55,7 +56,7 @@ private static void addReplaceOperationForOutcomeIfAttended(final IntegrationCon
 
         patchOperations.add(new ReplaceOperation(of(TARGET_OUTCOME_FIELD_NAME), valueOf(outcomeType)));
 
-        if ( notifyBehaviour ) {
+        if ( notifyBehaviour || NON_ATTENDANCE_VALUE.equalsIgnoreCase(attended)) {
```

Comment:
If this logic to notify on non-attended appointments applies to all appointments should the logic go into the Delius API?

Reply:
Does the "Refer to Office Manager" alert apply to all clients where an appointment id non-attended? I don't know how to answer this question. I will put this question to Andrew